src/output/icons.rs

```diff
@@ -205,9 +211,12 @@ const FILENAME_ICONS: Map<&'static str, char> = phf_map! {
     "Cargo.lock"          => Icons::LANG_RUST,      // 
     "Cargo.toml"          => Icons::LANG_RUST,      // 
     "CMakeLists.txt"      => Icons::MAKE,           // 
-    "composer.json"       => Icons::LANG_PHP,       // 
-    "composer.lock"       => Icons::LANG_PHP,       // 
+    "composer.json"       => Icons::LANG_PHP,       // 
+    "composer.lock"       => Icons::LANG_PHP,       // 
```

Comment:
again, not sure about these

Reply:
I agree with @cafkafk even tho those are useful files in php if not in a php context those icons don't make any sense.

Reply:
Seeing as there are two diff options for php icons.. perhaps use one for composer and another for .php files?